composer.json

```diff
@@ -14,32 +14,34 @@
         }
     ],
     "require": {
-        "php": ">=7.0.8",
+        "php": "^7.3|^8.0",
         "auto1-oss/service-api-components-bundle": "^1.0",
         "auto1-oss/service-api-request": "^1.0",
-        "symfony/serializer" : "~3.0|~4.0|~5.0",
-        "symfony/security-core" : "~3.0|~4.0|~5.0",
-        "symfony/security-csrf" : "~3.0|~4.0|~5.0",
-        "symfony/security-guard" : "~3.0|~4.0|~5.0",
-        "symfony/security-http" : "~3.0|~4.0|~5.0",
-        "symfony/monolog-bridge": "~3.0|~4.0|~5.0",
-        "symfony/dependency-injection": "~3.0|~4.0|~5.0",
-        "symfony/config": "~3.0|~4.0|~5.0",
-        "symfony/http-kernel": "~3.0|~4.0|~5.0",
-        "symfony/property-info": "^3.4.2|~4.0|~5.0",
-        "monolog/monolog": "~1.22",
+        "symfony/serializer" : "~3.0|~4.0|~5.0|~6.0",
```

Comment:
According to cockpit all the services using this library are at least on symfony 4.0, so I would propose to drop the "support" for 3.0